main.py

```diff
@@ -13,7 +13,7 @@
 def reg_trainer():
     ''' Regression Task Trainer '''
     config = RegConfig
-    X, Y = create_data_reg(train_size=config.train_size)
+    X, Y = create_data_reg(train_size=config.train_size, gap=True)
```

Comment:
Shouldn't this be set to False as a default?

Reply:
moved this into configuration file.